routers/api/v1/admin/user.go

```diff
@@ -136,6 +137,16 @@ func CreateUser(ctx *context.APIContext) {
 	}
 	log.Trace("Account created by admin (%s): %s", ctx.Doer.Name, u.Name)
 
+	// Back-date the user creation.
+	if form.Created != nil {
```

Comment:
I think this should be moved before line 124. For xorm inserting, it will check whether `created` is null, if it's not, it will be insert into the database.

Reply:
I tried that, but that didn't work; it would always be set to "now". I assumed that there was some "if creation → set creation timestamp" logic, which is why it's now done in a 2nd step. If you know a way to make this work with the initial creation query, that would indeed be better.